gadget.Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN cd /gadget/gadget-container && make gadget-container-deps
 # - https://github.com/kinvolk/traceloop/actions
 # - https://hub.docker.com/r/kinvolk/traceloop/tags
 
-FROM docker.io/kinvolk/traceloop:202109280218336f4efc as traceloop
+FROM docker.io/kinvolk/traceloop:202111051236207432a2 as traceloop
```

Comment:
The URL line 26 will need to be updated too (once https://github.com/kinvolk/traceloop/pull/64#pullrequestreview-798947657 is merged)